inf-terraform-aws/Jenkinsfile.template

```diff
@@ -3,7 +3,7 @@
 @Library('ods-jenkins-shared-library@@shared_library_ref@') _
 
 odsComponentPipeline(
-  imageStreamTag: '@ods_namespace@/jenkins-agent-terraform-2306:@agent_image_tag@',
```

Comment:
This Quickstarter is managed by the Testing team, it is better if we do not modify it if not required and as it have not been updated to use the new agent we should left it as it is. I will inform them so they can check and update based on the updated AWS Quickstarter